src/status_im/contexts/wallet/events.cljs

```diff
@@ -328,26 +328,54 @@
  (fn [{:keys [db]}]
    {:db (assoc db :wallet/valid-ens-or-address? false)}))
 
-(rf/reg-event-fx :wallet/get-address-details-success
- (fn [{:keys [db]} [{:keys [hasActivity]}]]
-   {:db (assoc-in db
-         [:wallet :ui :watch-address-activity-state]
-         (if hasActivity :has-activity :no-activity))}))
+(rf/reg-event-fx
```

Comment:
hmm, so this file is really starting to bloat.
@OmarBasem, @briansztamfater & I discussed this week of further splitting the wallet into sub-contexts
Mostly based off the figma keys, but we can have others that better fit.
<img width="189" alt="Screenshot 2024-02-29 at 18 30 20" src="https://github.com/status-im/status-mobile/assets/22799766/70b085dd-245d-4a3e-8a2a-866824a29ce0">

e.g something like
```
src/status_im/contexts/wallet/
     |-/accounts
         |-/events.cljs
         |-/view.cljs
         |-/style.cljs
      |-/add-account
         |-/events.cljs
         |-/view.cljs
         |-/style.cljs
         |-/create-account
         |-/add-account-to-watch
```
etc..
We have this for Send & Collectibles already too.

Anyway, I'm not saying to do this refactor here, but perhaps we can at least create the appropriate ns for these events added and move them to that file and the others can be moved in a refactor.

Also maybe these events are common enough to wallet that they should be in this ns. 🤷‍♂️

Reply:
@J-Son89 
Totally agree!

Actually I noticed this while adding these events, but noticed it's only happening on Collectibles and wasn't sure.
But yeah, I'll move them. mirrorring figma structure is a nice idea :+1: